src/compose/ComposeBox.js

```diff
@@ -372,6 +374,14 @@ class ComposeBox extends PureComponent<Props, State> {
         this.messageInputRef.current.focus();
       }
     }
+    if (nextProps.replyWithMention !== this.props.replyWithMention) {
+      const message = nextProps.replyWithMention ? nextProps.replyWithMention.content : '';
+      this.setMessageInputValue(message);
+      if (this.messageInputRef.current !== null) {
```

Comment:
We don't need the `!== null` here, we can just do `if (this.messageInputRef.current)`, or even better, no if statement and `this.messageInputRef.current?.focus();` should work.